Dokumentationen/test/test_cases.adoc

```diff
@@ -37,33 +37,34 @@ a| * Datenbankeintrag für die Bestellung ist angelegt
 
 |===
 
-|*Beschreibung* |Der Dozent versucht bewusst falsche Eingaben in den Voreinstellungen zu treffen. Damit sind keine menschlichen Fehler gemeint, wie z.B. ein falsches Datum ausgewählt zu haben, sondern ein Datum in der Vergangenheit zu wählen. 
+|*Beschreibung* |Der Dozent versucht bewusst falsche Eingaben in den Voreinstellungen zu treffen. Falsche Eingaben sind ein Datum in der Vergangenheit oder eine fehlerhafte Werte für die Zeit.
 
 
 |*Vorbedingungen und benötigte Daten*
 a| * ein Dozent ist mit validen Zugangsdaten angemeldet
 * es wurde eine Buchung begonnen
-* invalides Datum, invalide Uhrzeit und invalider Studiengang
+* es muss ein Dozentenkonto geben
```

Comment:
Ich habe gesehen, dass du das an mehreren Stellen hinzugefügt hast. Ich bin mir allerdings nicht sicher, ob wir das drinnen lassen sollen, da Prof. Anke in der Evaluation zu unserem vorherigen Beleg kommentiert hatte, ob sich das nicht aus der ersten Vorbedingung ergibt. Was sagst du dazu?